gym_graph_traffic/envs/intersection.py

```diff
@@ -26,10 +38,10 @@ def add_exit(self, dir_char, section) -> None:
     def finalize(self) -> None:
         raise NotImplementedError
 
-    def can_i_go(self, from_idx) -> int:
+    def can_i_go(self, segment, cars_indices) -> int:
```

Comment:
It's not clear to me what this method does now just by looking at function signature

Reply:
Should the second argument be "from_idx" to match the implementation?